zlib/Makefile

```diff
@@ -0,0 +1,37 @@
+PYODIDE_ROOT=$(abspath ..)
+include ../Makefile.envs
+
+ZLIBVERSION=1.2.11
```

Comment:
There is already this same version available under `pyodide/cpython/build/zlib-1.2.11/` could you link to it instead? BTW, it's also in the emscripten ports usable via `-s USE_ZLIB=1` flag (see how it's done in matplotlib `meta.yaml` file)

Reply:
@rth I tried to link with an existing zlib directory in the project, but I couldn't..

libxml2 expects a directory with `include` and `lib`, see: https://github.com/GNOME/libxml2/blob/mainline/configure.ac#L193

That's why I had to make install zlib to its own build directory.

It is possible to build libxml2 without zlib support, however I am not sure if this affects the usage of lxml.